airflow/bin/cli.py

```diff
@@ -362,6 +362,10 @@ def run(args, dag=None):
     task = dag.get_task(task_id=args.task_id)
     ti = TaskInstance(task, args.execution_date)
     ti.refresh_from_db()
+    # Don't allow `ariflow run --raw to uplod` to S3/GCS, the `airflow run
```

Comment:
nit: typoe, should be `ariflow run --raw` to uplooad

Reply:
good spot, thanks